app/services/businesses_search_service.rb

```diff
@@ -1,11 +1,21 @@
 class BusinessesSearchService
   def self.utility_search(location, type)
     response = conn.get("/#{location}/utilities/#{type}")
+    parsed(response)
+  end
+
+  def self.recreation_search(location, type)
+    response = conn.get("/#{location}/recreation/#{type}")
     JSON.parse(response.body, symbolize_names: true)
```

Comment:
This should be moved to to the parsed method as well

Reply:
100%! We will make sure we do that with the next iteration when we push up the recreation branch!